docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Virtual Background with Effects SDK
+nav: 1.45
+---
+
+<Note type="warning">
+    This feature is still in Beta. To know more or report any issues, feel free to reach out to us
+    over <a href={'https://discord.com/invite/kGdmszyzq2'}>Discord</a>.
+</Note>
+
+
+## Pre-requisites
+
+**Get the 100ms VirtualBackground Package**
+
+```bash section=GetHMSVirtualBackgroundPackage sectionIndex=1
+npm install --save @100mslive/hms-virtual-background@latest
```

Comment:
Let us also mention the version from which this is supported

Reply:
Added